lib/bolt/pal.rb

```diff
@@ -141,6 +141,14 @@ def register_resource_types(loaders)
       end
     end
 
+    def detect_project_conflict(project, modules)
+      return unless project
+      if modules.any? { |mod| mod.name == project.name }
+        Bolt::Logger.warn_once("project shadows module",
+                               "The project '#{project.name}' will shadow an existing module of the same name")
```

Comment:
```suggestion
                               "The project '#{project.name}' shadows an existing module of the same name")
```

I think removing the tense makes this read a little clearer and avoids any potential confusion around 'will shadow' being in the future.